middleware/2022-11-02-1.3.41.md

```diff
@@ -0,0 +1,61 @@
+---
+authors: poscreator
+slug: middleware/1.3.41
+tags: [Middleware, Germany]
+---
+
+# Middleware 1.3.41 (Germany)
+In this version of the Middleware, we have increased the default fiskaly timeout to enable longer TAR export to be performed, we have also reworked the Android launcher and the Swissbit SCU. And  we have increased the default timeout of the helipad helper as well as added a new configuration parameter regarding the readcount for the journalDE entries.
+Additionally, we have added ARMEL support to our SQLite packages. And finally we have created a script which helps generating indexes for MySQL databases used in a bring your own data center environment.
```

Comment:
With the change above and Paul's comment in slack, I think that would not be needed anymore 😅